<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_login">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton_login{
	selection-color: rgb(0, 170, 127);
	background-color:rgb(82, 156, 113);
	border-radius:20px;
	font: 16pt &quot;SentyTEA 新蒂下午茶体&quot;;
	color: rgb(255, 255, 255);
}
#pushButton_login{
	padding_top:20px;
}
</string>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUser">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>211</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:1px;
font: 9pt &quot;01FLOPDESIGN&quot;;
color:rgb(59, 118, 87);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>user</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditPassword">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>262</y>
      <width>160</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:1px;
font: 9pt &quot;01FLOPDESIGN&quot;;
color:rgb(59, 118, 87);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>password</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>70</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/image/resourse/头像（1）.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>260</width>
      <height>380</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(125, 200, 158);
border-radius:20px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="edit_background_1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="edit_background_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>200</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:15px</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>214</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/resourse/头像.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>56</x>
      <y>266</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/image/resourse/钥匙.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="Exit">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>30</y>
      <width>26</width>
      <height>26</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:10px;
background-color: rgb(255, 255, 255);
font: 9pt &quot;01FLOPDESIGN&quot;;</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>111</width>
      <height>18</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>记住账号密码</string>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>edit_background_2</zorder>
   <zorder>edit_background_1</zorder>
   <zorder>pushButton_login</zorder>
   <zorder>lineEditUser</zorder>
   <zorder>lineEditPassword</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>Exit</zorder>
   <zorder>checkBox</zorder>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>42</y>
    </hint>
    <hint type="destinationlabel">
     <x>150</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
